component/authenticator/src/test/java/org/wso2/carbon/identity/authenticator/smsotp/test/SMSOTPAuthenticatorTest.java

```diff
@@ -745,15 +822,18 @@ public void testGetScreenAttribute() throws UserStoreException, AuthenticationFa
         when(realmService.getTenantUserRealm(-1234)).thenReturn(userRealm);
         when(userRealm.getUserStoreManager()).thenReturn(userStoreManager);
         when(userRealm.getUserStoreManager()
-                .getUserClaimValue("admin", "http://wso2.org/claims/mobile", null)).thenReturn("0778965231");
+                .getUserClaimValue("admin", "http://wso2.org/claims/mobile", null)).thenReturn(
+                "0778965231");
         when(SMSOTPUtils.getNoOfDigits(context)).thenReturn("4");
 
         // with forward order
-        Assert.assertEquals(smsotpAuthenticator.getScreenAttribute(context,userRealm,"admin"),"0778******");
+        Assert.assertEquals(smsotpAuthenticator.getScreenAttribute(context, userRealm, "admin"),
+                "0778******");
 
         // with backward order
         when(SMSOTPUtils.getDigitsOrder(context)).thenReturn("backward");
-        Assert.assertEquals(smsotpAuthenticator.getScreenAttribute(context,userRealm,"admin"),"******5231");
+        Assert.assertEquals(smsotpAuthenticator.getScreenAttribute(context, userRealm, "admin"),
```

Comment:
don't format the places which doesn't have any changes.

Reply:
Addressed in [648dbd5](https://github.com/wso2-extensions/identity-outbound-auth-sms-otp/pull/181/commits/648dbd564d6bb53434f20449c67d03a281250d2b) and [b49a3f5](https://github.com/wso2-extensions/identity-outbound-auth-sms-otp/pull/181/commits/b49a3f5adca772f88265cf948d64f227e2b85675)